packages/regl-worldview/src/Worldview.js

```diff
@@ -104,6 +108,7 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
         cameraState: props.cameraState || props.defaultCameraState || DEFAULT_CAMERA_STATE,
         onCameraStateChange: props.onCameraStateChange || undefined,
       }),
+      dragStartPos: null,
```

Comment:
we don't have to put this in the component `state` if it's not going to be used in the `render` method. Let's just keep it as a class property instead.